ui/src/datasource/DatasourceOverview.vue

```diff
@@ -110,41 +117,55 @@
 <script lang="ts">
 import Vue from 'vue'
 import Component from 'vue-class-component'
-
+import {
+  convertMillisecondsToHours,
+  convertMillisecondsToMinutes,
+  getISOTimeString,
+  getISODateString
+} from '../helpers/date-helpers'
 import Datasource from './datasource'
 import * as DatasourceREST from './datasourceRest'
 
-const ONE_HOUR_IN_MS = 3600 * 1000
-const ONE_MINUTE_IN_MS = 60 * 1000
-
-@Component({})
+@Component({
+  methods: { getISOTimeString, getISODateString }
```

Comment:
Interesting! Never saw it like that. How is it different from just importing and using it?

Reply:
This way, we can call the helpers directly from the template without having an additional method in the component class that just forwards the call.